game-authorization-score/src/main/java/com/iconbet/score/authorization/Authorization.java

```diff
@@ -0,0 +1,965 @@
+	private final VarDB<Address> super_admin = Context.newVarDB(SUPER_ADMIN, Address.class);
+	//  question=?  why adrress
+	private final DictDB<Address,String> proposal_data = Context.newDictDB(PROPOSAL_DATA, String.class);
+	private final DictDB<Address,String> status_data = Context.newDictDB(STATUS_DATA, String.class);
+	private final DictDB<Address,Address> owner_data = Context.newDictDB(OWNER_DATA, Address.class);
+	private final ArrayDB<Address> proposal_list = Context.newArrayDB(PROPOSAL_LIST, Address.class);
+	private final VarDB<BigInteger> day = Context.newVarDB(DAY, BigInteger.class);
+	private final DictDB<Address, BigInteger[]> wagers = Context.newDictDB(WAGERS, BigInteger[].class);
+	private final DictDB<Address, BigInteger[]> payouts = Context.newDictDB(PAYOUTS, BigInteger[].class);
+	
+	private final VarDB<BigInteger> game_developers_share = Context.newVarDB(GAME_DEVELOPERS_SHARE, BigInteger.class);
+	private final DictDB<Address,BigInteger> todays_games_excess = Context.newDictDB(TODAYS_GAMES_EXCESS, BigInteger.class);
+	
+	private final VarDB<BigInteger> new_div_changing_time = Context.newVarDB(NEW_DIV_CHANGING_TIME, BigInteger.class);
+	private final DictDB<Address, BigInteger[]> games_excess_history = Context.newDictDB(TODAYS_GAMES_EXCESS, BigInteger[].class);
+
+	private final VarDB<Boolean> apply_watch_dog_method = Context.newVarDB(APPLY_WATCH_DOG_METHOD, Boolean.class);
+	private final DictDB<Address,BigInteger> maximum_payouts = Context.newDictDB(MAXIMUM_PAYOUTS, BigInteger.class);
+	private final VarDB<BigInteger> maximum_loss = Context.newVarDB(MAXIMUM_LOSS, BigInteger.class);
+
+	  
+
+	@Override
+	public String name() {
+		return null;
+	}
+	@Override
+	public String symbol() {
+		return null;
+	}
+	@Override
+	public int decimals() {
+		return 0;
+	}
+	@Override
+	public BigInteger totalSupply() {
+		return null;
+	}
+	@Override
+	public BigInteger balanceOf(Address _owner) {
+		return null;
+	}
+	
+	@Override
+	@EventLog(indexed=2)
+	public void transfer(Address to, BigInteger value, byte[] data) {}
+	
+	@Override
+	@EventLog(indexed=2)
+	public void Transfer(Address from, Address to, BigInteger value, byte[] data) {}
+	
+	@EventLog(indexed=2)
+	public void ProposalSubmitted(Address sender, Address scoreAddress) {}
+	
+	@EventLog(indexed=1)
+	public void GameSuspended(Address scoreAddress, String note) {}
+
+	
+	@External
+	public void untether() {
+		/*
+        A function to redefine the value of self.owner once it is possible.
+        To be included through an update if it is added to IconService.
+
+        Sets the value of self.owner to the score holding the game treasury.
+		 */
+		//Context.getOrigin() - > txn.origin  - always wallet
+		//Context.getCaller() - > sender
+		//Context.getOrigin() - > owner
+		if (!Context.getOrigin().equals(Context.getOwner()))
+			Context.revert("Only the owner can call the untether method.");
+	}
+	
+	
+	@External
+	public void set_new_div_changing_time(BigInteger timestamp) {
```

Comment:
Not sure if we need to preserve the parameter name, we need to test it.
lets say if we convert it to http request... 

http://localhost/set_new_div_changing_time?timestamp=10000000000000000 
is different than
http://localhost/set_new_div_changing_time?_timestamp=10000000000000000

Reply:
Updated variable names with original names.